doc/manuals/devel/README.md

```diff
@@ -28,5 +28,6 @@ Also, as NGSI is used for the payload of the requests, some previous knowledge o
 * [Source code: main program and libraries](sourceCode.md). A description of the different libraries (and main program) in which the source code is structured.
 * [Flow index](flowsIndex.md). An index for all the flow diagrams described in the development documentation. A very useful "map" to have at hand.
 * [Semaphores](semaphores.md). This document provides detailed information about the different semaphores that Orion uses for internal synchronization.
+* [Http Headers](httpHeaders.md). This document provides list of all Http Headers used by Orion along with their description.
```

Comment:
```suggestion
* [HTTP Headers](httpHeaders.md). This document provides list of all HTTP headers used by Orion along with their description.
```

I think that in general we use HTTP instead of Http along documentation, so better to use the later one. Please, review httpHeaders.md in the case more cases need to be fixed.

Reply:
@ArqumFarooqui-NEC whenever you fix a comment raised in the PR please add a `Fixed in <githash>` in that comment thread. That we the reviewer (me :) can know when all the comments have been addressed so the PR gets ready to re-review.

More detail on the "pull request" protocol used in Orion here: https://github.com/telefonicaid/fiware-orion/blob/master/doc/manuals/contribution_guidelines.md#pull-request-protocol

Thanks!

Reply:
> I think that in general we use HTTP instead of Http along documentation, so better to use the later one. Please, review httpHeaders.md in the case more cases need to be fixed.

fixed in 0855ee9